autopts/pybtp/defs.py

```diff
@@ -55,19 +55,20 @@ def BIT(bit):
 BTP_SERVICE_ID_PBP = 0x1e
 # GENERATOR append 1
 
+BTP_ERROR = 0x00
 BTP_STATUS_SUCCESS = 0x00
 BTP_STATUS_FAILED = 0x01
 BTP_STATUS_UNKNOWN_CMD = 0x02
 BTP_STATUS_NOT_READY = 0x03
 BTP_STATUS = 0x00
-CORE_READ_SUPPORTED_COMMANDS = 0x01
-CORE_READ_SUPPORTED_SERVICES = 0x02
-CORE_REGISTER_SERVICE = 0x03
-CORE_UNREGISTER_SERVICE = 0x04
-CORE_LOG_MESSAGE = 0x05
-CORE_EV_IUT_READY = 0x80
-GAP_READ_SUPPORTED_COMMANDS = 0x01
-GAP_READ_CONTROLLER_INDEX_LIST = 0x02
+BTP_CMD_CORE_READ_SUPPORTED_COMMANDS = 0x01
+BTP_CMD_CORE_READ_SUPPORTED_SERVICES = 0x02
+BTP_CMD_CORE_REGISTER_SERVICE = 0x03
+BTP_CMD_CORE_UNREGISTER_SERVICE = 0x04
+BTP_CMD_CORE_LOG_MESSAGE = 0x05
+BTP_EV_CORE_IUT_READY = 0x80
```

Comment:
What is the rationale for `BTP_CMD_<SERVICE>` and `BTP_EV_<SERVICE>` compared to `BTP_<SERVICE>_CMD` and `BTP_<SERVICE>_EV?

Reply:
actually,  I just notice this comment and TBH I think we should do BTP_<SERVICE>_CMD and BTP_<SERVICE>_EV